spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -904,6 +923,13 @@ def setup_menu(self):
         )
         self.copy_action.setShortcut(keybinding('Copy'))
         self.copy_action.setShortcutContext(Qt.WidgetShortcut)
+        self.histogram_action = self.create_action(
+            name=DataframeEditorActions.Histogram,
+            text=_("Histogram"),
```

Comment:
```suggestion
            text=_("Histogram"),
            tip=_("Plot a histogram of the selected columns")
```

I think this is necessary to tell users how the button works. Please edit my suggestion if you think it can be described better.